api/external/data_feed/data_feed.proto

```diff
@@ -36,6 +76,19 @@ service DatafeedService {
 			action: "destination:destination:delete"
 		};
 	};
+/*
+  Update a datafeed destination
+
+  Updates a datafeed destination.
```

Comment:
@satellite15 Does this action update the datafeed destination ID or other details that does not include the datafeed destination ID? 

Related: Can you update more than one detail at a time with this action?

Reply:
^ I still have these two questions, but I edited this content with my current interpretation, which can always be changed.

Reply:
You can modify more than one detail, I just tested modifying the id and it doesn't work.